lib/carrierwave/uploader/versions.rb

```diff
@@ -263,19 +300,14 @@ def full_original_filename
       end
 
       def cache_versions!(new_file)
-        dependent_versions.each do |name, v|
+        versions_to_cache.each do |name, v|
           v.send(:cache_id=, @cache_id)
           v.cache!(new_file)
         end
       end
 
-      def store_versions!(new_file, versions=nil)
```

Comment:
Now versions to store is part of object's state, so we don't need extra argument here.